cfme/containers/provider/__init__.py

```diff
@@ -110,8 +110,8 @@ class ContainersProvider(BaseProvider, Pretty):
         'num_replication_controller',
         'num_pod',
         'num_node',
-        'num_container',
-        'num_image']
+        'num_image_registry',
+        'num_container']
```

Comment:
I thought I'd override that function, by taking out num_image for now. Reason being, num_image works for 5.6 but not for 5.7, and I haven't made any changes in the API related to images. Should I? I added num_image_registry which was not there before. I have added the Template object to the API, let me know what you think of this

https://github.com/ManageIQ/wrapanapi/pull/96